test/Microsoft.TemplateEngine.Cli.UnitTests/AddProjectsToSolutionPostActionTests.cs

```diff
@@ -101,5 +101,68 @@ public void AddProjectToSolutionPostActionDoesntFindProjectOutOfRange()
             Assert.False(AddProjectsToSolutionPostAction.TryGetProjectFilesToAdd(EngineEnvironmentSettings, postAction, creationResult, string.Empty, out IReadOnlyList<string> foundProjectFiles));
             Assert.Null(foundProjectFiles);
         }
+
+        [Fact(DisplayName = nameof(AddProjectToSolutionPostActionFindsMultipleProjectsToAddWithOutputBasePath))]
```

Comment:
Apparently, there is a difference: the default display name contains also the namespace of the test "Microsoft.TemplateEngine.Cli.UnitTests.".

Reply:
Interesting, thanks. I wonder where the fully qualified name would cause issues. Or where it would surface at all, since Test Explorer is evidently showing bare method names even without `DisplayName`:

![image](https://user-images.githubusercontent.com/12206368/92227859-32f3a980-eea7-11ea-9e97-92aad1cb0600.png)

Anyway, just a random comment, feel free to ignore. Thanks!